examples/model_merge_example/server.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+from collections import OrderedDict
+from functools import partial
+from pathlib import Path
+from typing import Any, Dict, Optional, Sequence, Tuple
+
+import flwr as fl
+import torch
+import torch.nn as nn
+from flwr.common import Config, NDArrays, Scalar
+from flwr.server.client_manager import SimpleClientManager
+from torch.utils.data import DataLoader
+
+from examples.models.cnn_model import MnistNet
+from fl4health.server.model_merge_server import ModelMergeServer
+from fl4health.strategies.model_merge_strategy import ModelMergeStrategy
+from fl4health.utils.config import load_config
+from fl4health.utils.load_data import load_mnist_data
+from fl4health.utils.metric_aggregation import evaluate_metrics_aggregation_fn, fit_metrics_aggregation_fn
+from fl4health.utils.metrics import Accuracy, Metric, MetricManager
+
+
+def evaluate_fn(
+    model: nn.Module,
+    loader: DataLoader,
+    metrics: Sequence[Metric],
+    device: torch.device,
+    round: int,
+    parameters: NDArrays,
+    config: Config,
+) -> Optional[Tuple[float, Dict[str, Scalar]]]:
+    model.to(device)
+    model.eval()
+    evaluate_metric_manager = MetricManager(metrics, "evaluate")
+
+    params_dict = zip(model.state_dict().keys(), parameters)
+    state_dict = OrderedDict({k: torch.tensor(v) for k, v in params_dict})
+    model.load_state_dict(state_dict, strict=True)
+
+    with torch.no_grad():
+        for input, target in loader:
+            input, target = input.to(device), target.to(device)
+            preds = {"predictions": model(input)}
+            evaluate_metric_manager.update(preds, target)
+
+    return 0.0, evaluate_metric_manager.compute()
+
+
+def main(config: Dict[str, Any], data_path: Path, batch_size: int) -> None:
+    _, val_loader, _ = load_mnist_data(data_path, batch_size)
```

Comment:
The reason I did this is because I wanted the client side evaluation dataset to be different than the server side evaluation set. Do you think its worth it or not? I can change it to be the same if you think thats better

Reply:
Nope. That's good with me! Maybe just put a comment there to short circuit such a question?